packages/blade/docs/tokens/Colors.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, DocsContainer, Story, ColorPalette, ColorItem } from '@storybook/addon-docs';
+import { useTheme, BladeProvider } from '@razorpay/blade/components';
+import { paymentTheme, bankingTheme, colors } from '@razorpay/blade/tokens';
+import flat from 'flat';
+import get from 'lodash/get';
+
+<Meta
+  title="Tokens/Colors"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokens === 'paymentTheme') {
+            return paymentTheme;
+          }
+          if (context.globals.themeTokens === 'bankingTheme') {
+            return bankingTheme;
+          }
+        };
+        return (
+          <DocsContainer context={context}>
+            <BladeProvider
+              key={`${context.globals.themeTokens}-${context.globals.colorScheme}`}
+              themeTokens={getThemeTokens()}
+              colorScheme={context.globals.colorScheme}
+            >
+              {children}
+            </BladeProvider>
+          </DocsContainer>
+        )
+      }
+    },
+  }}
+/>
+
+
+# 🎨 Color Palette
+
+## Chromatic Colors
+
+<ColorPalette>
+{Object.entries(colors.chromatic).map(([category, swatches]) => (
```

Comment:
Some of these colors have a very horrible horizontal scroll :/

No way to make it wrap to next line?

Reply:
no 😢